example/styles.css

```diff
@@ -160,9 +160,6 @@ button:not(.cf-btn) {
 }
 
 .cf-card__footer_message {
```

Comment:
Why? `.cf-card__footer_message` shouldn't even be used anymore since `<Text/>` relies on a different set of CSS classes right?